components/Nav/NavPrimary.module.css

```diff
@@ -19,26 +19,30 @@
 
 .navListItem {
   list-style: none;
+  margin: 2px;
+  margin-left: 10px;
 }
 
 .navListItem a {
   text-decoration: none;
-  padding: 16px;
-  display: block;
+  padding: 4px;
+  display: inline-block;
+  border-radius: 5px;
 }
 
 .navListItem a:hover {
   background-color: var(--primaryLt);
 }
 
 .navList a:focus {
-  background-color: var(--primaryLt);
-  font-weight: bold;
+  background-color: var(--primary);
+  color: var(--white);
```

Comment:
Setting the text to white means the color contrast on dark mode doesn't work. Please use the --text variable instead. Please also switch the background color back to primaryLt